pynsee/geodata/_get_geodata.py

```diff
@@ -173,12 +173,13 @@ def _get_geodata(
             data_all = _geojson_parser(json).reset_index(drop=True)
 
         else:
-            msg = "!!! Query is correct but no data found !!!"
-            print(msg)
-            print("Query:\n%s" % link)
+            msg = "Query is correct but no data found !"
+            logger.error(msg)
+            logger.debug("Query:\n%s" % link)
```

Comment:
I'd use error for the query too since we'd want users to give us as much detail about the error message as possible